lib/airbrake-ruby/backtrace.rb

```diff
@@ -71,5 +81,16 @@ def stack_frame(match)
           function: match[:function] }
       end
     end
+
+    def self.match_frame(regexp, stackframe)
+      match = regexp.match(stackframe)
+      return match if match
+
+      unless (matched_frame = GENERIC_STACKFRAME_REGEXP.match(stackframe))
+        raise Airbrake::Error, "failed parsing '#{stackframe}'"
+      end
+
+      matched_frame
```

Comment:
You have `match` and `matched_frame`. Also your function looks following

```
return match
raise error
return match
```

but would be nice if it looks like

```
return match
return match
raise error
```